src/components/cylc/GraphSubgraph.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <g v-if="subgraph.label!='margin'" class="c-graph-subgraph">
+    <rect
+      :width="subgraph.width"
+      :height="subgraph.height"
+      :x="subgraph.x"
+      :y="subgraph.y"
+      rx="50"
+      ry="50"
+      fill="none" />
+    <text
+      :x="calcLabelXPosition"
+      :y="calcLabelYPosition"
+      font-family="Roboto"
+      alignment-baseline="middle" text-anchor="middle"
+      font-size=70
```

Comment:
This font size should match the one defined in the dot file to ensure that graphviz leaves the right amount of space for the label that we draw.

But note that the units may differ so you might need to specify `px` or `mm` as appropriate (I'm not sure what units GraphViz uses by default, I know dimensions are in inches so fonts might be odd).

Reply:
SVG default is pixels but accepts cm, mm, in, em, ex, pt, pc, and px

Graphviz uses points (pt)
https://graphviz.org/doc/info/attrs.html
but it seems happy enough when given pixels